### ISSUE TITLE
Could you please provide datasets with subdivision connectivity

### ISSUE THREAD

**Issue by liang3588** [lzhengning/SubdivNet#6]
I found the datasets download via "sh scripts/<DATASET_NAME>/get_data.sh" are original datasets without the subdivision connectivity. Moreover, it is very hard to remesh them by datagen_maps.py for the remeshing is always interrupted by some problems in the program, especially for the Manifold40.

Therefore, could you please provide the remeshed datasets with subdivision connectivity? Thank you very much!

**Comment by lzhengning** [comment]
Hi @liang3588,

Can you tell me which dataset you have downloaded are the original ones, Or all of them are?

**Comment by liang3588** [comment]
I only download the ModelNet40, and it is the original one. When I want to train the model, the program reports an error, indicating that there is no Manifold40-MAPS-96-3.

**Comment by liang3588** [comment]
"unzip -q Manifold40.zip && rm Manifold40.zip" Maybe only ModelNet40 has this problem.

**Comment by lzhengning** [comment]
Sorry that I mixed up with the downloading scripts. I  have updated the scripts for downloading Manifold40 with subdivision connectivity. 

BTW, current `datagen_maps.py`  is imperfect, and may fail due to the reasons I listed at head of the scripts. For example, in ModelNet40, the genus of meshes is ranged from zero to hundreds. Therefore some complicated shapes are hard to be remeshed. But we can take a special care of such meshes, with different configurations of MAPS and the manifold tool. The common solution is also listed in the scripts.
